langchain4j-couchbase/src/test/java/dev/langchain4j/store/embedding/couchbase/Utils.java

```diff
@@ -0,0 +1,70 @@
+package dev.langchain4j.store.embedding.couchbase;
+
+import com.couchbase.client.java.Bucket;
+import com.couchbase.client.java.Cluster;
+import dev.langchain4j.model.embedding.EmbeddingModel;
+import dev.langchain4j.model.embedding.onnx.allminilml6v2q.AllMiniLmL6V2QuantizedEmbeddingModel;
+import org.testcontainers.couchbase.BucketDefinition;
+import org.testcontainers.couchbase.CouchbaseContainer;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+import org.testcontainers.shaded.com.google.common.base.Suppliers;
+import org.testcontainers.utility.DockerImageName;
+
+import java.time.Duration;
+import java.util.function.Supplier;
+
+@Testcontainers
+public final class Utils {
```

Comment:
The ambiguous class name and everything public - is no go 😞  try an interface with default implementations instead

Reply:
made package-private and renamed the class to `CouchbaseTestUtils`, will that work?

Reply:
not perfect, but fine for me 😄